### PR TITLE
Feat/unit info panel

### PR DESCRIPTION
**Pull request by JeroenVinke**
- [ ] Code review
- [ ] Comments
- [ ] Functional tests written
- [ ] Functional tests checked by someone other than the author(s)
- [ ] No linting errors
- [ ] Unit tests passes
- [ ] Technical documentation written
- [ ] Technical documentation reviewed
- [ ] Removed unnecessary includes

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@
 #include "sdl/label/manager/description_manager.h"
 #include "sdl/badge/sdl_badge_render_object.h"
 #include "camera/camera_manager.h"
+#include <sdl/panel/sdl_unit_info_panel.h>
+#include <sdl/label/sdl_render_resource_label.h>
```
Comment (MarkVlamPersie) [inline src/main.cpp]: inlcudes with "..." instead of <>

File: src/ui/sdl/panel/sdl_stacked_panel.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+private:
+    Orientation _orientation;
+public:
+    SDLStackedPanel(SDL_RenderObject *, Orientation orientation);
+
+    void render(SDLRenderer *renderer, mat2 &m, float d) override;
+
+    void add_component(SDLPanel *child);
+};
+
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/sdl_stacked_panel.h]: destructor?

File: src/ui/sdl/panel/sdl_unit_info_panel.cpp [hunk 3]
```diff
@@ -0,0 +1,65 @@
+//
+// Created by Jeroen on 6/1/2017.
+//
+
+#include "sdl/sdl_render_object.h"
+#include "entity/player.h"
+#include "entity/player_manager.h"
+#include "settings.h"
+#include <string>
+#include <iostream>
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/sdl_unit_info_panel.cpp]: iostream still needed?

File: src/ui/sdl/panel/unit_info/sdl_unit_info_entity.cpp [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+//
+
+#include <SDL_ttf.h>
+#include <string>
+#include <map>
+#include "sdl/label/sdl_render_label.h"
+#include <iostream>
+#include "sdl/image/sdl_image_render_object.h"
+#include "settings.h"
+#include <entity/static/building/building_entity.h>
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/unit_info/sdl_unit_info_entity.cpp]: "..." ipv <...>

File: src/ui/sdl/panel/unit_info/sdl_unit_info_entity.cpp [hunk 5]
```diff
@@ -0,0 +1,66 @@
+//
+// Created by Jeroen on 6/6/2017.
+//
+
+#include <SDL_ttf.h>
+#include <string>
+#include <map>
+#include "sdl/label/sdl_render_label.h"
+#include <iostream>
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/unit_info/sdl_unit_info_entity.cpp]: still needed? dont see any cout's.

File: src/ui/sdl/panel/unit_info/sdl_unit_info_entity.h [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    std::map<std::string, SDLRenderLabel *> _labels;
+public:
+    SDLUnitInfoEntity(MovingEntity* entity, SDL_RenderObject *r);
+
+    SDLUnitInfoEntity(BuildingEntity *entity, SDL_RenderObject *r);
+
+    SDLUnitInfoEntity(BaseEntity *entity, SDL_RenderObject *r, std::string texture);
+
+    void render(SDLRenderer *renderer, mat2 &m, float d) override;
+};
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/unit_info/sdl_unit_info_entity.h]: destructor?

File: src/ui/sdl/panel/sdl_unit_info_panel.h [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+class BuildingEntity;
+class MovingEntity;
+class SDLStackedPanel;
+
+class SDLUnitInfoPanel : public SDLPanel {
+private:
+    std::string _previous_units_hash;
+
+    BuildingEntity *_previous_building;
+
```
Comment (MarkVlamPersie) [inline src/ui/sdl/panel/sdl_unit_info_panel.h]: destructor?

File: src/renderer/sdl/label/sdl_render_resource_label.h [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    Resources *_resource;
+
+    ResourceType _rt;
+
+    const char *get_resource_text();
+public:
+    SDLRenderResourceLabel(const vec2 &position, const vec2 &size, sdl_data *data, std::string, ResourceType, TTF_Font *);
+
+    void render(SDLRenderer *renderer, const mat2 &) override;
+};
```
Comment (MarkVlamPersie) [inline src/renderer/sdl/label/sdl_render_resource_label.h]: destructor?

File: src/ui/sdl/panel/sdl_stacked_panel.h (hunk 2 above)
Comment (Rboin) [inline src/ui/sdl/panel/sdl_stacked_panel.h]: Destructor is not needed here, since this class doesn't hold a pointer to anything.

**Pull request merged**